help/develop/admin/deploy/cloudformation/cloudformation.yml

```diff
@@ -765,6 +883,65 @@ Resources:
       - Key: stickiness.lb_cookie.duration_seconds
         Value: 86400
     Type: AWS::ElasticLoadBalancingV2::TargetGroup
+  GrokComputeTaskDefinitionEc2:
```

Comment:
Why do you need a copy of TaskDefinition?

Reply:
done